code/game/machinery/doors/airlock.dm

```diff
@@ -1562,6 +1641,15 @@
  * Arguments:
  * * frame_state - The AIRLOCK_FRAME_ value, as used in update_overlays()
  **/
+
+/obj/machinery/door/airlock/proc/note_type() //Returns a string representing the type of note pinned to this airlock
+	if(!note)
+		return
+	else if(istype(note, /obj/item/paper))
+		return "note"
+	else if(istype(note, /obj/item/photo))
+		return "photo"
+
 /obj/machinery/door/airlock/proc/get_note_state(frame_state)
```

Comment:
Bro you shifted a doc comment over what are you DOING
(also this proc doesn't seem to be used, why was it added?)

Reply:
fixed, unnecessary change by me